tests/manage/monitoring/prometheus/test_deployment_status.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import tier4
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@pytest.mark.polarion_id("OCS-1052")
+def test_ceph_manager_stopped(workload_stop_ceph_mgr):
+    """
+    Test that there is appropriate alert when ceph manager
+    is unavailable and that this alert is cleared when the manager
+    is back online.
+    """
+    prometheus = PrometheusAPI()
+
+    # get alerts from time when manager deployment was scaled down
+    alerts = workload_stop_ceph_mgr.get('prometheus_alerts')
+    target_label = 'CephMgrIsAbsent'
+    target_msg = 'Storage metrics collector service not available anymore.'
+    target_alerts = [
+        alert
+        for alert
+        in alerts
+        if alert.get('labels').get('alertname') == target_label
+    ]
+    log.info(f"Checking properties of found {target_label} alerts")
+    msg = f"Incorrect number of {target_label} alerts"
+    assert len(target_alerts) == 2, msg
+
+    msg = 'Alert message is not correct'
+    assert target_alerts[0]['annotations'] == target_msg, msg
+
+    msg = 'First alert doesn\'t have warning severity'
+    assert target_alerts[0]['annotations']['severity_level'] == 'warning', msg
+
+    msg = 'First alert is not in pending state'
+    assert target_alerts[0]['state'] == 'pending', msg
+
+    msg = 'Alert message is not correct'
+    assert target_alerts[1]['annotations'] == target_msg, msg
+
+    msg = 'Second alert doesn\'t have warning severity'
+    assert target_alerts[1]['annotations']['severity_level'] == 'warning', msg
+
+    msg = 'First alert is not in firing state'
+    assert target_alerts[1]['state'] == 'firing', msg
+
+    # seconds to wait before alert is cleared after measurement is finished
+    time_min = 30
+
+    time_actual = time.time()
+    time_sleep = int(
+        (workload_stop_ceph_mgr.get('stop') + time_min) - time_actual
+    )
+    if time_sleep > 0:
```

Comment:
please create a function to wait for alerts - you can use it everywhere instead of sleep

Reply:
done